lib/katello/tasks/repository.rake

```diff
@@ -78,6 +78,71 @@ namespace :katello do
     handle_manifest_list_label_updates
   end
 
+  desc "Clean up duplicate erratum_packages"
+  task :cleanup_duplicate_erratum_packages => ["dynflow:client", "check_ping"] do
+    User.current = User.anonymous_api_admin
+    handle_duplicate_erratum_packages
+  end
+
+  def handle_duplicate_erratum_packages
+    # Alert users that they need to reindex their database to ensure the indexes are re-run and active.
+    puts "Please reindex your database to ensure indexes are rebuilt and active."
+    puts "This can be achieved by running `runuser -u postgres -- reindexdb -a`"
+    # Get all duplicate groups
```

Comment:
```suggestion
    puts "This can be achieved by running `runuser -u postgres -- reindexdb -a` as root."
    puts "Seek technical support for any errors with database reindexing."
    # Get all duplicate groups
```